clang/test/CodeGen/aarch64-neon-vcmla.c

```diff
@@ -1,444 +1,596 @@
-// RUN: %clang_cc1 -triple arm64-apple-ios -target-feature +neon \
-// RUN:        -target-feature +v8.3a \
-// RUN:        -target-feature +fullfp16 \
-// RUN:        -disable-O0-optnone -emit-llvm -o - %s | opt -S -O1 | FileCheck %s
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 -triple arm64-linux-gnu -target-feature +neon  \
```

Comment:
Why was the run line changed here ?

Reply:
Just an artifact of recreating that test file - though I have reverted this file back to it's previous state following our resolution of `vcmla_laneq_f16`, which was the reason this was originally changed at all.